azext_iot/_params.py

```diff
@@ -879,6 +892,18 @@ def load_arguments(self, _):
             options_list=["--auth-type"],
             arg_type=hub_auth_type_dataplane_param_type,
         )
+        context.argument(
+            "custom_metric_queries",
+            nargs="+",
+            options_list=["--custom-metric-queries", "--cmq"],
+            help="An alternative way of input style(narg key-value pair) to --metrics and intended to replace it in the future.",
+        )
+        context.argument(
+            "custom_labels",
+            nargs="+",
+            options_list=["--custom-labels", "--cl"],
+            help="An alternative way of input style(narg key-value pair) to --labels and intended to replace it in the future.",
```

Comment:
The description should mention the delimiter, it should say space separated key-value pairs. Also why not add a conceptual usage example in the arg help description as well. We do it in other places i.e. Example: key=value or key="value value".

Reply:
Thanks, help text updated!